src/site/content/en/blog/display-override/index.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: Preparing for the display modes of tomorrow
+subhead: PWAs can use the "display_override" property to deal with special display modes.
+authors:
+  - thomassteiner
+date: 2021-02-16
+description: |
+  The display_override property allows developers to define a customized fallback chain of modes
+  their PWAs should be displayed in.
+hero: hero.jpg
+alt: Web App Manifest source code excerpt.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - progressive-web-apps
```

Comment:
What about creating a web app manifest tag? Do we have more than two articles?

Reply:
Added in https://github.com/GoogleChrome/web.dev/pull/4675/commits/1032982659136da400ff5eba2a032655ccccead2.